schemas/constructs/v1alpha3/selectors.json

```diff
@@ -45,90 +46,103 @@
           },
           "patch": {
             "type": "object",
-            "additionalProperties": false,
-            "properties": {
-              "patchStrategy": {
-                "type": "string",
-                "enum": [
-                  "replace"
-                ]
+            "allOf": [
+              {
+                "properties": {
+                  "patchStrategy": {
```

Comment:
```suggestion
                  "patchStrategy": {
                  "description": "patchStrategy allows you to make specific changes to a resource using a standard JSON Patch format (RFC 6902). The Strategic Merge Patch is specific to Kubernetes and understands the structure of Kubernetes objects. It can handle complex changes like updating lists and maps, as well as preserving default values. However, it's not supported for custom resources. For custom resources, only JSON Patch and Merge Patch are typically supported."
```

Reply:
look for the JSON merge startegies.